notebooks/CubevizExample.ipynb

```diff
@@ -61,11 +68,15 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "fn = download_file('https://stsci.box.com/shared/static/28a88k1qfipo4yxc4p4d40v4axtlal8y.fits', cache=True)\n",
+    "data_dir = tempfile.gettempdir()\n",
+    "#data_dir = \"/User/username/Data/\"\n",
+    "\n",
+    "fn = \"jw02732-c1001_t004_miri_ch1-longmediumshort-_s3d.fits\"\n",
+    "result = Observations.download_file(f\"mast:JWST/product/{fn}\", local_path=f'{data_dir}/{fn}')\n",
```

Comment:
You should not have to include filename in the path?
```suggestion
    "result = Observations.download_file(f\"mast:JWST/product/{fn}\", local_path=f'{data_dir}')\n",
```

Reply:
I agree that you shouldn't, but when I tested without it failed (contrary to the `Observations` docs).

Reply:
Hmmmm!

Reply:
Oh, a known bug: https://github.com/astropy/astroquery/issues/2501

Reply:
yes I think it's supposed to be a path rather than a filename.